targettests/Remote-Sim/free_func.cpp

```diff
@@ -21,7 +19,11 @@ void ghz(std::size_t N) __qpu__ {
   cudaq::qvector q(N);
   h(q[0]);
   for (int i = 0; i < N - 1; i++) {
+#if CUDAQ_USE_STD20
     x<cudaq::ctrl>(q[i], q[i + 1]);
+#else
+    cx(q[i], q[i + 1]);
+#endif
```

Comment:
It's better to make this 2 distinct tests. The C++20 variation will only work with C++20 and so on. But the C++17 version must work with C++17, C++20, C++23, ...

Reply:
Good point. Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1435/commits/4ad7ecd84a4f0d7c8dac820095463a5234365331